docker/build_matrix.yaml

```diff
@@ -43,97 +43,116 @@
   PYTORCH_VERSION: 1.12.1
   TAGS:
   - mosaicml/pytorch:1.12.1_cu116-python3.9-ubuntu20.04
-  - mosaicml/pytorch:latest
   TARGET: pytorch_stage
   TORCHTEXT_VERSION: 0.13.1
   TORCHVISION_VERSION: 0.13.1
-- BASE_IMAGE: nvidia/cuda:11.6.2-cudnn8-devel-ubuntu20.04
-  CUDA_VERSION: 11.6.2
+- BASE_IMAGE: ubuntu:20.04
+  CUDA_VERSION: ''
+  MOFED_VERSION: ''
   PYTHON_VERSION: '3.9'
   PYTORCH_VERSION: 1.12.1
   TAGS:
-  - mosaicml/pytorch_vision:1.12.1_cu116-python3.9-ubuntu20.04
-  - mosaicml/pytorch_vision:latest
-  TARGET: vision_stage
+  - mosaicml/pytorch:1.12.1_cpu-python3.9-ubuntu20.04
+  TARGET: pytorch_stage
   TORCHTEXT_VERSION: 0.13.1
   TORCHVISION_VERSION: 0.13.1
+- BASE_IMAGE: nvidia/cuda:11.7.1-cudnn8-devel-ubuntu20.04
+  CUDA_VERSION: 11.7.1
+  PYTHON_VERSION: '3.10'
+  PYTORCH_VERSION: 1.13.0
+  TAGS:
+  - mosaicml/pytorch:1.13.0_cu117-python3.10-ubuntu20.04
+  - mosaicml/pytorch:latest
+  TARGET: pytorch_stage
+  TORCHTEXT_VERSION: 0.14.0
+  TORCHVISION_VERSION: 0.14.0
+- BASE_IMAGE: nvidia/cuda:11.7.1-cudnn8-devel-ubuntu20.04
+  CUDA_VERSION: 11.7.1
+  PYTHON_VERSION: '3.10'
+  PYTORCH_VERSION: 1.13.0
+  TAGS:
+  - mosaicml/pytorch_vision:1.13.0_cu117-python3.10-ubuntu20.04
+  - mosaicml/pytorch_vision:latest
+  TARGET: vision_stage
+  TORCHTEXT_VERSION: 0.14.0
+  TORCHVISION_VERSION: 0.14.0
 - BASE_IMAGE: ubuntu:20.04
   CUDA_VERSION: ''
   MOFED_VERSION: ''
-  PYTHON_VERSION: '3.9'
-  PYTORCH_VERSION: 1.12.1
+  PYTHON_VERSION: '3.10'
+  PYTORCH_VERSION: 1.13.0
   TAGS:
-  - mosaicml/pytorch:1.12.1_cpu-python3.9-ubuntu20.04
+  - mosaicml/pytorch:1.13.0_cpu-python3.10-ubuntu20.04
   - mosaicml/pytorch:latest_cpu
   TARGET: pytorch_stage
-  TORCHTEXT_VERSION: 0.13.1
-  TORCHVISION_VERSION: 0.13.1
+  TORCHTEXT_VERSION: 0.14.0
+  TORCHVISION_VERSION: 0.14.0
 - BASE_IMAGE: ubuntu:20.04
   CUDA_VERSION: ''
   MOFED_VERSION: ''
-  PYTHON_VERSION: '3.9'
-  PYTORCH_VERSION: 1.12.1
+  PYTHON_VERSION: '3.10'
+  PYTORCH_VERSION: 1.13.0
   TAGS:
-  - mosaicml/pytorch_vision:1.12.1_cpu-python3.9-ubuntu20.04
+  - mosaicml/pytorch_vision:1.13.0_cpu-python3.10-ubuntu20.04
   - mosaicml/pytorch_vision:latest_cpu
   TARGET: vision_stage
-  TORCHTEXT_VERSION: 0.13.1
-  TORCHVISION_VERSION: 0.13.1
-- BASE_IMAGE: nvidia/cuda:11.6.2-cudnn8-devel-ubuntu20.04
+  TORCHTEXT_VERSION: 0.14.0
+  TORCHVISION_VERSION: 0.14.0
+- BASE_IMAGE: nvidia/cuda:11.7.1-cudnn8-devel-ubuntu20.04
   COMPOSER_INSTALL_COMMAND: mosaicml[all]
-  CUDA_VERSION: 11.6.2
-  PYTHON_VERSION: '3.9'
-  PYTORCH_VERSION: 1.12.1
+  CUDA_VERSION: 11.7.1
+  PYTHON_VERSION: '3.10'
+  PYTORCH_VERSION: 1.13.0
```

Comment:
Same thing regarding all the `mosaicml/composer` images, lets leave them to 1.12 for now.  For testing we can just pull the 1.13 images and install composer manually.